tests/integration/API/PostFile.test.ts

```diff
@@ -0,0 +1,63 @@
+/* tslint:disable:no-relative-imports */
+import { asyncPipe } from 'Helpers/asyncPipe'
+import * as fs from 'fs'
+import * as path from 'path'
+import { prop, identity, head, map } from 'ramda'
+import { describe } from 'riteway'
+
+import { FileHelper } from '../../helpers/files'
+import { IPFS } from '../../helpers/ipfs'
+import { runtimeId, setUpServerAndDb } from '../../helpers/utils'
+
+const PREFIX = `test-functional-node-poet-${runtimeId()}`
+const NODE_PORT = '28081'
+
+const ipfs = IPFS()
+const fileHelper = FileHelper({ port: NODE_PORT  })
+
+const getHash = prop('hash')
+const getStatus = prop('status')
+
+const calcPath = (relPath: string) => path.resolve(__dirname, relPath)
+
+const files = {
+  utf8Markdown: calcPath('../../fixtures/files/utf8.md'),
+}
+
+/* tslint:disable-next-line:max-line-length */
```

Comment:
Is this disable needed?

Reply:
I think this one was needed, but I will double check.

Reply:
Nope, it wasn't needed, guess it was my imagination.